gradle/libs.versions.toml

```diff
@@ -1,15 +1,15 @@
 [versions]
 libautoupdate = "1.3.1"
-moulconfig = "3.5.0"
+moulconfig = "3.6.0"
```

Comment:
why do we bump neu version? is this relevant to this pr? it got not mentioned in the pr desc. if this is relevant, please explain why and mention it in the changelog technical details. if its not. remove the change.

Reply:
this version supports 1.21.5, i looked at the diff between the last one and it has no changes for legacy

Reply:
What else did the version bump do?

Reply:
you can look at it if you want https://github.com/NotEnoughUpdates/MoulConfig/compare/3.5.0...3.6.0 . To me it just looks like some restructuring and adding 1.21.5 support

Reply:
@nea89o can you confirm there are no chagnes for the 1.8.9 version of moulconfig?

@NopoTheGamer please include that version bump in the changelog

Reply:
There are no changes intended between those two versions. What prompted you to think so?

Reply:
the fact that preemptive moulconfig version bumps in the past introduced new bugs to sh. and since the version bump was not mentioned or explained in this pr